perun-core/src/main/java/cz/metacentrum/perun/core/impl/Auditer.java

```diff
@@ -533,18 +505,52 @@ public int getAuditerMessagesCount(PerunSession perunSession) throws InternalErr
 	}
 
 	/**
-	 * Stores the auditer messages to the DB in batch.
+	 * Stores the list of auditer messages to the DB in batch.
+	 *
+	 * It also checks if there are any messages which can be resolved by registered attribute modules.
+	 * Store these resolved messages too.
 	 *
-	 * @param messages list of AuditerMessages
+	 * @param auditerMessages list of AuditerMessages
 	 */
-	public void storeMessagesToDb(final List<AuditerMessage> messages) {
+	public void storeMessagesToDb(final List<AuditerMessage> auditerMessages) {
+		//Avoid working with empty list of auditer-messages
+		if(auditerMessages == null || auditerMessages.isEmpty()) {
+			log.error("Trying to store empty list of messages to DB!");
```

Comment:
Error level seems to be exaggerated. I would use trace level, because calling this method with empty list seems to be common. For example during commit of transaction with no associated audit messages.

Reply:
Ok.